lib/ansible/modules/cloud/cloudstack/cs_instance_facts.py

```diff
@@ -227,6 +233,18 @@ def get_instance(self):
                         break
         return self.instance
 
+    def get_volumes(self, instance):
+        volume_details = []
+        if instance:
+            args                = {}
+            args['projectid']   = instance['projectid']
```

Comment:
also add account and domain id e.g.
~~~python
args['account'] = self.get_account(key='name')
args['domainid'] = self.get_domain(key='id')
~~~